content/en/blog/falcoctl-gitlab-oci-artifacts-support.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "GitLab Container Registry Now Supports falcoctl OCI Artifact mediaTypes"
+linktitle: "GitLab Container Registry Now Supports falcoctl OCI Artifact mediaTypes"
+date: 2023-08-11
+author: Batuhan Apaydın
+slug: gitLab-container-registry-now-supports-falcoctl-oci-artifact-mediaTypes
+tags: ["Rules","Falco Plugins","Falcoctl", "GitLab", "OCI", "OCI Artifacts"]
```

Comment:
```suggestion
title: ""GitLab Container Registry now supports Falcoctl OCI Artifacts"
linktitle: ""GitLab Container Registry now supports Falcoctl OCI Artifacts"
date: 2023-08-11
author: Batuhan Apaydın
slug: gitLab-supports-falcoctl-ociartifacts
tags: ["Rules", "Falco Plugins", "Falcoctl", "Configuration Management"]
```

Reply:
May of the included tags wouldn't be supported by the Falco Blog. [Here](https://github.com/falcosecurity/falco-website/blob/master/data/en/blog_tags.yaml) is the supported list.

Reply:
thank you!